src/js/utils/styles.js

```diff
@@ -21,6 +21,7 @@ export const backgroundStyle = (background, theme) => {
       }
       return css`
         background: ${background.image} no-repeat center center;
```

Comment:
We should remove the `center center` here, since we are always setting `background-position`.